tests/test_pipdeptree.py

```diff
@@ -488,3 +490,16 @@ def test_custom_interpreter(tmp_path, monkeypatch, capfd, args_joined):
     assert context.value.code == 1
     assert not out
     assert err == "graphviz functionality is not supported when querying" " non-host python\n"
+
+
+def test_guess_version_setuptools_not_imported(monkeypatch):
+    builtin_import = builtins.__import__
+
+    def _import(name, *args, **kwargs):
+        if name in {"importlib.metadata", "importlib_metadata"}:
+            raise ImportError
+        return builtin_import(name, *args, **kwargs)
+
+    with monkeypatch.context() as m:
```

Comment:
use pytes-tmock instead of monkeypatch - https://github.com/tox-dev/pipdeptree/blob/main/pyproject.toml#L25

Reply:
Done